support/Machines/Sonic.yaml

```diff
@@ -0,0 +1,14 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+Machine:
+  Name: Sonic
+  Description: |
+    HPC at ICTS-TIFR hosted by AstroRel.
+    More information:
+    https://it.icts.res.in/docs/sonic-cluster/
+  DefaultTasksPerNode: 6
+  DefaultProcsPerTasks: 16
```

Comment:
Please make these consistent between the machine file and the submit script.

Reply:
Thanks for catching this.